Client/Frontend/Settings/SettingsNavigationController.swift

```diff
@@ -33,10 +33,23 @@ extension ThemedNavigationController: Themeable {
         navigationBar.isTranslucent = false
         navigationBar.tintColor = UIColor.theme.general.controlTint
         navigationBar.titleTextAttributes = [NSAttributedString.Key.foregroundColor: UIColor.theme.ecosia.highContrastText]
+        navigationBar.shadowImage = nil
+        navigationBar.setBackgroundImage(UIImage(), for: .default)
+        navigationBar.shadowImage = UIImage()
         setNeedsStatusBarAppearanceUpdate()
         viewControllers.forEach {
             ($0 as? Themeable)?.applyTheme()
         }
+        
+        let separator = UIView()
+        separator.backgroundColor = UIColor.theme.ecosia.barSeparator
+        separator.translatesAutoresizingMaskIntoConstraints = false
+        navigationBar.addSubview(separator)
```

Comment:
won't this be added multiple times with every call to `applyTheme()`? 🤔 I think that can be called quite often, so I'd see a chance to rather put the separator in a property which is only added once.

Reply:
Good point, please check the revision